imap/cyr_expire.c

```diff
@@ -650,22 +849,20 @@ static int parse_args(int argc, char *argv[], struct arguments *args)
 int main(int argc, char *argv[])
 {
     int r = 0;
-    struct arguments args;
+    struct cyr_expire_ctx ctx;
```

Comment:
@elliefm `gcc` on travis doesn't accept struct intialisations with `{0}`, so I'm continuing to use `memset` for now.